inspirehep/dojson/hep/fields/bd01x09x.py

```diff
@@ -171,15 +155,15 @@ def _is_handle(type_, id_):
                         type_ = 'HDL'
                     persistent_identifiers.append({
                         'source': source,
-                        'type': type_,
+                        'schema': type_,
                         'value': id_,
                     })
 
     self['dois'] = dois
     return persistent_identifiers
 
 
-@hep2marc.over('024', '^(dois|persistent_identifiers)$')
+@hep2marc.over('024', '^persistent_identifiers$')
 def dois2marc(self, key, value):
```

Comment:
you should change the function name here to `persistent_identifiers2marc` as it is not about dois anymore, and you have another `dois2marc` later.